error[E0119]: conflicting implementations of trait `Unpin` for type `Foo<_, _>`
  --> tests/ui/not_unpin/conflict-unpin.rs:5:15
   |
5  | #[pin_project(!Unpin)] //~ ERROR E0119
   |               ^^^^^^ conflicting implementation for `Foo<_, _>`
...
12 | impl<T, U> Unpin for Foo<T, U> where T: Unpin {}
   | --------------------------------------------- first implementation here
   |
   = note: upstream crates may add a new impl of trait `std::marker::Unpin` for type `_::_pin_project::__private::Wrapper<'_, std::marker::PhantomPinned>` in future versions

error[E0119]: conflicting implementations of trait `Unpin` for type `Bar<_, _>`
  --> tests/ui/not_unpin/conflict-unpin.rs:14:15
   |
14 | #[pin_project(!Unpin)] //~ ERROR E0119
   |               ^^^^^^ conflicting implementation for `Bar<_, _>`
...
21 | impl<T, U> Unpin for Bar<T, U> {}
   | ------------------------------ first implementation here
   |
   = note: upstream crates may add a new impl of trait `std::marker::Unpin` for type `_::_pin_project::__private::Wrapper<'_, std::marker::PhantomPinned>` in future versions

error[E0119]: conflicting implementations of trait `Unpin` for type `Baz<_, _>`
  --> tests/ui/not_unpin/conflict-unpin.rs:23:15
   |
23 | #[pin_project(!Unpin)] //~ ERROR E0119
   |               ^^^^^^ conflicting implementation for `Baz<_, _>`
...
30 | impl<T: Unpin, U: Unpin> Unpin for Baz<T, U> {}
   | -------------------------------------------- first implementation here
   |
   = note: upstream crates may add a new impl of trait `std::marker::Unpin` for type `_::_pin_project::__private::Wrapper<'_, std::marker::PhantomPinned>` in future versions
